src/layout/default.hbs

```diff
@@ -7,5 +7,6 @@
     {{{body}}}
     <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
```

Comment:
I mentioned this earlier, but you should  only need one of these jQuery scripts, unless I'm confused about what they are. I would drop the `1.10.2` and put the `2.1.3` above your `index.js`